README.md

```diff
@@ -1 +1,199 @@
-# release-notes-generator-action
\ No newline at end of file
+<h1>
+  <p align="center">
+    Release Note Generator GitHub Action
+  </p>
+</h1>
+<p align="center">
+  This repository provides a GitHub action to <strong>automatically create a release notes</strong> when a milestone is closed.
+</p>
+
+**Table of Contents**
+  - [Common usage](#common-usage)
+  - [Why?](#why)
+  - [Startup](#startup)
+    - [Use GitHub action](#use-github-action)
+      - [Settings for v2.0.0+ release](#settings-for-v200-release)
+      - [NOTE](#note)
+      - [Result illustration](#result-illustration)
+    - [Configure output folder](#configure-output-folder)
+    - [Custom output release file](#custom-output-release-file)
+      - [Prefixed name using v2.0.0+](#prefixed-name-using-v200)
+    - [Use Milestone title](#use-milestone-title)
+      - [Prefixed name using v2.0.0+](#prefixed-name-using-v200-1)
+      - [Result illustration](#result-illustration-1)
+  - [How to use the generated file](#how-to-use-the-generated-file)
+
+## Common usage
+
+Each time a milestone is closed, this GitHub action scans its attached issues and pull request, and automatically generates a wonderful release note.  
+
+_The action uses [Spring.io changelog generator](https://github.com/spring-io/github-changelog-generator) tool._
+
+<p align="center">
+  <img src="https://github.com/step-security/release-notes-generator-action/raw/main/images/release_notes.png" alt="Result illustration"/>
+<p>
+
+## Why?
+
+Why aren't we simply automating the release creation and push the release notes there?  
+We think in some cases it is better to keep release notes separated from the release process... yeah, maybe the right name is not release notes anymore. :) Taking as example a continuous deployment project: __anytime you push to the repository, a new version of the application is built and pushed in production.__ And so, a new release is created to complete the process; a release with a **single** feature inside.  
+We think it is not a good idea to push to 'users' a new release notes message every time you generate a changes in the source code (let us say 10 times a day?).  
+It may be a better way to communicate only once every week or once the sprint is over, which gives you a clearer and complete release notes containing the aggregated information pushed in production within the last sprint (or any days).
+
+## Startup
+
+### Use GitHub action
+
+#### Settings for v2.0.0+ release
+
+Create a file into your root project directory: `.github/workflows/release-notes.yml`:
+```yaml
+# Trigger the workflow on milestone events
+on: 
+  milestone:
+    types: [closed]
+name: Milestone Closure
+jobs:
+  create-release-notes:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Create Release Notes
+      uses: docker://step-security/release-notes-generator-action:2.0.1
```

Comment:
We should update the documentation to use step-security/release-notes-generator-action@v3. We can discuss more in a meeting